### PR TITLE
[MTSRE-669] removing pullSecret from metadata schema

### PR DESCRIPTION
**Pull request by Ankit152**
Signed-off-by: Ankit Kurmi <akurmi@redhat.com>

# py-mtcli

## Description

Removing `pullSecret` from `metadata.schema.yaml`.

### CODE REVIEW

File: tests/sss/test_catalogue_source.py [hunk 1]
```diff
@@ -62,8 +62,6 @@ def test_pull_secret_injection():
         assert name is not None
         assert data is not None
         assert data["spec"]["image"] is not None
-        assert len(data["spec"]["secrets"]) == 1
```
Comment (yashvardhan-kukreja) [inline tests/sss/test_catalogue_source.py:65]: this entire test can be removed

Comment (Ankit152) [reply tests/sss/test_catalogue_source.py:65]: Done 🙂

**Review by Ajpantuso — APPROVED** [review]
LGTM

**Pull request merged**